### PR TITLE
Update from Intelsdi-x pluginsync 0.1.6

### PR DESCRIPTION
**Pull request by nanliu**

### CODE REVIEW

**Comment by sandlbn** [comment]
Looks like problem with cross compiling on the linux for darwin.  C.AI_MASK doesn't exist on the linux. Maybe we should drop support for MacOS. I think there's not too many libvirt/darwin users .

**Comment by nanliu** [comment]
Thanks for the info. I'm going to drop the darwin build from the matrix for now.

File: .travis.yml [hunk 1]
```diff
@@ -1,25 +1,75 @@
...
-    - TEST=unit
+  - TEST_TYPE=small
+  - TEST_TYPE=build
+matrix:
+  exclude:
+  - go: 1.6.3
+    env: TEST_TYPE=build
 before_install:
-- sudo apt-get update -y  
-- sudo apt-get install libvirt-bin libvirt-dev -y
```
Comment (kindermoumoute) [inline .travis.yml]: I'm surprised it's building without this line.

Comment (nanliu) [inline .travis.yml]: because addon:

```
+  addons:
 +    apt:
 +      packages:
 +        - libvirt-bin
 +        - libvirt-dev
```

**Pull request merged**